codewars11.py

```diff
@@ -0,0 +1,10 @@
+# https://www.codewars.com/kata/52597aa56021e91c93000cb0/train/python
+
+def move_zeros(array):
+    zero_count = [value for value in array if value == 0]
+    array = [value for value in array if value != 0]
+    array.extend(zero_count)
+
+    return array
+
+print(move_zeros([1, 0, 0, 2]))
```

Comment:
Did you come up with this yourself? This is brilliant solution and very efficient! Amazing job!